tests/acceptance/features/bootstrap/WebDavPropertiesContext.php

```diff
@@ -1275,7 +1276,12 @@ public function theseEtagsShouldNotHaveChanged(TableNode $etagTable):void {
 			$user = $this->featureContext->getActualUsername($user);
 			$actualEtag = $this->getCurrentEtagOfElement($path, $user);
 			$storedEtag = $this->getStoredEtagOfElement($path, $user, __METHOD__);
-			if ($actualEtag !== $storedEtag) {
+			if ($shouldShouldNot === 'should' && $actualEtag === $storedEtag) {
+				$changedEtagCount = $changedEtagCount + 1;
+				$changedEtagMessage
+					.= "\nThe etag '$storedEtag' of element '$path' of user '$user' did not change.";
+			}
+			if ($shouldShouldNot === 'should not' && $actualEtag !== $storedEtag) {
 				$changedEtagCount = $changedEtagCount + 1;
```

Comment:
```suggestion
		        $changedEtagCount = $changedEtagCount + 1;
			if ($shouldShouldNot === 'should' && $actualEtag === $storedEtag) {
				$changedEtagMessage
					.= "\nThe etag '$storedEtag' of element '$path' of user '$user' did not change.";
			}
			if ($shouldShouldNot === 'should not' && $actualEtag !== $storedEtag) {
```

Reply:
Moving `$changedEtagCount = $changedEtagCount + 1;` outside of the if block will fail while asserting.

Reply:
Ahh! double if. I thought nested if

Reply:
Should we keep this way or revert it to original? @saw-jan